tfe/resource_tfe_organization.go

```diff
@@ -78,6 +78,11 @@ func resourceTFEOrganization() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"allow_force_delete_workspaces": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
Since the API produces the default value, you may want to remove Default and set Computed to true. My understanding of Default is that we use it when we want the provider to set a state value, perhaps to backfill a new required field? I'm going to smoke test both cases, but the other settings in this config seem to rely on the API defaults so it would be strange to have a new argument with slightly different behavior.